### PR TITLE
basic ticker seeding function

### PR DESCRIPTION
**Pull request by bellcd**

### CODE REVIEW

**Comment by mgoyke** [comment]
Works perfectly, well done, and thank you.

**Comment by mgoyke** [comment]
I don't have write access to do the merge, not sure if I am supposed to do that after verification or not.

File: seeds/tickers.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+// TODO: add test suite
+
+function createTickers() {
+  // returns the same array of 100 unique strings (tickers)
+    // each ticker is 3 - 4 characters in length // TODO: vary the ticker length
+    // each ticker is made up of the letters A - Z
+
+    let seed = 1;
```
Comment (LesliePajuelo) [inline seeds/tickers.js]: Why is the seed 1? can you add a context comment please. Or what is the effect of changing this value?

File: seeds/tickers.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+        // every time the incrementor changes, build the ticker in a different order
+        if (incrementor === 0) {
+          ticker.push(letter);
+        } else if (incrementor === 1) {
+          ticker.unshift(letter);
+        } else if (incrementor === 2) {
+          ticker.splice(Math.floor(ticker.length / 2), 0, letter);
+        } else {
+          if (incrementor === 3) {
```
Comment (LesliePajuelo) [inline seeds/tickers.js]: Can you make this a function so it's more generic and less repetitive.

**Pull request merged**